README.md

```diff
@@ -260,7 +260,7 @@ once every day, at midnight, the `TimeInstant` reported for each measurement wil
 such measurements, the `TimeInstant` attribute would take those values around midnight.
 
 This functionality can be turned on and off through the use of the `timestamp` configuration flag (described in the
-configuration).
+configuration), as well ass 'timestamp' flag in device or group provision.
```

Comment:
ass -> as

Reply:
fixed in 5d2d3f3cfa54d855bef864cd2efd8c8c52d7eb85